src/data.js

```diff
@@ -2064,6 +2064,27 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Muhammad Umair',
+    description:
+      'Fullstack front-end developer and designer, passionate in everything #JavaScript',
+    url: 'https://www.linkedin.com/in/mumairofficial',
```

Comment:
I think there is one more issue here. This [`uses.tech`](https://uses.tech/) site actually shows a list of `/uses` pages detailing developer setups, gear, software and configs like:

```
https://wesbos.com/uses
https://farrant.me/uses
https://codewithhugo.com/uses/
```

but the URL you have shared here is just your LinkedIn profile. Please add your `/uses` page url here, if you have any. Thanks!

Reply:
I second this, setting this to "request changes"

Reply:
@HugoDF I have created gist for now. Thank you for the idea; mean while I am developing my personal portfolio site!